software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java

```diff
@@ -314,6 +314,7 @@ public void launch() {
 
         if (isPackageInstall()) {
             commands.add(addSbinPathCommand());
+            commands.add(sudo(format("ulimit -n %s", getNumberOfOpenFiles())));
```

Comment:
This will increase the soft limit, but what about the hard limit? Distributions usually have a hard limit < 65536 so this will fail.